test/libsolidity/ASTJSONTest.cpp

```diff
@@ -92,6 +102,7 @@ ASTJSONTest::ASTJSONTest(string const& _filename)
 	string source;
 	string line;
 	string const delimiter("// ----");
+	string const failMarker("// Fail on:");
```

Comment:
How about we call it `failAfter` to make it easier to notice the connection with `stopAfter`?
```suggestion
	string const failMarker("// failAfter:");
```

Reply:
It makes sense.